.bob/skills/find-missing-variant-periodics/SKILL.md

```
@@ -0,0 +1,177 @@
+- **Exists**: Source files that already have target files
+
+Calculate statistics:
+- Total source files found
+- Number missing target files
+- Number with existing target files
+- Percentage missing
+
+### 6. Display results
+
+Show summary statistics:
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)